### ISSUE TITLE
Changing indentation on numbers and bullets

### ISSUE THREAD

**Issue by convergenceIM** [convergenceIM/alpha-scientist#36]
The theme seems to have weird indentations for numbered and bulleted lists.  See below images.  Can these be brought in so that the bullets (or numbers) align with the text and the text is slightly to the right? 

![image](https://user-images.githubusercontent.com/22535225/42909807-30fd0b26-8a9a-11e8-8258-69196216d6f9.png)

![image](https://user-images.githubusercontent.com/22535225/42909826-3ef0ba48-8a9a-11e8-85a6-8c96e27068e2.png)

**Comment by yn-coder** [comment]
You need to have a bullet and number sign arranged by test line?

**Comment by convergenceIM** [comment]
Exactly.  Looks weird to have the bullets sticking out past the text.  Maybe there's a way to do via markdown but I can't find it

**Comment by yn-coder** [comment]
No, it's css coded
